code/clue.js

```diff
@@ -1,59 +1,351 @@
+    image: "assets/library.png",
+  },
+  {
+    name: "Billiard room",
+    description:
+      "We enter the Billiard room. It's dark, woodpaneled room with a huge billiardtable in the middle. Of course, the body is found lying straight on the table. What a mess..",
+    image: "assets/billiard.png",
+  },
+  {
+    name: "Lounge",
+    description:
+      "We enter the Lounge. Velvet couches and a broken glasstable. Sherry is spilled on a dark, blue carpet. Wait a minute.. that's not sherry! A body is sticking out under one of the couches..",
+    image: "assets/lounge.png",
+  },
+  {
+    name: "Ballroom",
+    description:
+      "We enter the Ballroom. Once there was a ball filled with joy and music. Now there's only tragic, blood and death. The body is found in a corner of the room, the once so shiny wooden floor is now covered in blood.",
+    image: "assets/ballroom.png",
+  },
+  {
+    name: "Hallway",
+    description:
+      "We enter the Hallway. Creeky floorboards and portraits glaring. The killer must have been very quiet and have known exactly which boards to avoid to not be discovered. Behind a big plant we find the body, the former lady of the house sneering from her portrait.",
+    image: "assets/hall.png",
+  },
+  {
+    name: "Spa",
+    description:
+      "We enter the Spa. Soft music playing, the smell of chlorine and moist air. If it wasn't for the body floating in the pool, it would be quite inviting.",
+    image: "assets/spa.png",
+  },
+  {
+    name: "Living Room",
+    description:
+      "We enter the Living room. With the cracked tv's distorted sound in the background and whiskeybottle on the floor, there are plenty of evidence of a scuffle. The body was carefully placed in a chair in front of the tv.",
+    image: "assets/livingroom.png",
+  },
+  {
+    name: "Guest House",
+    description:
+      "We enter the guest house. It has a spartan decor with two small beds. This is far from inviting, black mold covering the corners of the ceiling. The body was found in the closet, maybe the murderer was in a rush?",
+    image: "assets/guesthouse.png",
+  },
+  {
+    name: "Patio",
+    description:
+      "This patio could really use a renovation. Loose floorboards everywhere. A perfect place to hide a body.",
+    image: "assets/patio.png",
+  },
+  {
+    name: "Observatory",
+    description:
+      "One would love to gaze at the stars here. Trinkets and books about space everywhere. Unfortunately also bloodspladder everywhere..",
+    image: "assets/observatory.png",
+  },
+];
 
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
+const randomSelector = (array) => {
+  return array[Math.floor(Math.random() * array.length)];
+};
 
-const suspects = [
-  mrGreen,
-  mrsWhite
-  // ...  and the rest
-]
+const mystery = {
+  killer: "",
+  weapon: "",
+  room: "",
+  favouriteWeapon: "",
```

Comment:
Once again - clean approach would be initialising your object with `null` values